composer.json

```diff
@@ -1,12 +1,16 @@
 {
-    "name": "damiantw/laravel-roles",
-    "description": "Role Based Authorization For Laravel 5",
+    "name": "dgeowils/laravel-roles",
+    "description": "Role Based Authorization For Laravel 6",
     "keywords": ["authorization", "laravel", "roles", "authorities"],
     "license": "MIT",
     "authors": [
         {
             "name": "Damian Crisafulli",
             "email": "damian.crisafulli@troyweb.com"
+        },
```

Comment:
you can remove this - originally i was going to fork and create my own distribution, but i think it makes more sense to keep everything with this repo